exonum-java-binding-common/src/test/java/com/exonum/binding/common/serialization/StandardSerializersTest.java

```diff
@@ -24,4 +30,55 @@ static void invalidBytesValueTest(byte[] invalidValue, Serializer serializer) {
     assertThrows(IllegalArgumentException.class, () -> serializer.fromBytes(invalidValue));
   }
 
+  static IntStream intValues() {
+    return IntStream.range(0, 32)
+        .map(value -> 1 << value)
+        .flatMap(value -> IntStream.of(-value, value - 1, value, value + 1));
+  }
+
+  private static LongStream longValues() {
```

Comment:
The methods used as sources _in other classes_ must not be private — that breaks IDEA (they are shown as unused).